src/w3c/headers.js

```diff
@@ -745,8 +745,11 @@ async function deriveHistoryURI(conf) {
   );
 
   // If it's on the Rec Track or it's TR worthy, then allow history override.
-  // Also make a an exception for FPWD.
-  if ((conf.historyURI && canShowHistory) || conf.specStatus === "FPWD") {
+  // Also make a an exception for FPWD, DNOTE, NOTE and DRY.
+  if (
+    (conf.historyURI && canShowHistory) ||
+    ["FPWD", "DNOTE", "NOTE", "DRY"].includes(conf.specStatus)
+  ) {
```

Comment:
These exceptions shouldn't be needed as `trStatus` includes them.

Reply:
these are actually special use cases. A new specification can start as a FPWD, DNOTE, NOTE or DRY. In this case, the history link doesn't exist yet so adding the exception will display it without checking if it exists or not.

Reply:
I'm not following. (Maybe I'm just sleepy)
1. As per line 742, `historyURI` can be a `respecConfig.historyURI` or one inferred via `shortName`.
2. `canShowHistory` is true for `DNOTE, NOTE, DRY` (they're part of `trStatus`).
   - `["FPWD", "DNOTE", "NOTE", "DRY"].includes(conf.specStatus)` is a subset of `canShowHistory`
4. When exceptions path is followed (i.e. if `conf.historyURI` is falsy), we end up setting `conf.historyURI = "https://www.w3.org/standards/history/{shortName}"`. Is that intended?

Reply:
Let's say you want to publish a FPWD and you expect respec to infer the history link so you don't add a `respecConfig.historyURI`.
In this case, without the exception, respec will do a `HEAD` to find the history link, which will return a 404 since it doesn't exist yet, and `conf.historyURI` will be empty.

Reply:
In that case, we'll end up having `"https://www.w3.org/standards/history/{shortName}"` as `History:` header link, right? Is that intended?

Unless author manually disables History link with `respecConfig.historyURI = null`, we'll have a non-existing history link for FPWD et. al?